runtime/near-vm-runner-standalone/src/script.rs

```diff
@@ -0,0 +1,222 @@
+use std::fs;
+use std::path::Path;
+
+use near_primitives::types::CompiledContractCache;
+use near_primitives_core::profile::ProfileData;
+use near_primitives_core::runtime::fees::RuntimeFeesConfig;
+use near_vm_logic::mocks::mock_external::MockedExternal;
+use near_vm_logic::types::PromiseResult;
+use near_vm_logic::{ProtocolVersion, VMConfig, VMContext, VMKind, VMOutcome};
+use near_vm_runner::{run_vm_profiled, MockCompiledContractCache, VMError};
+
+use crate::State;
+
+#[derive(Clone, Copy)]
+pub struct Contract(usize);
+
+/// Constructs a "script" to execute several contracts in a row. This is mainly
+/// intended for VM benchmarking.
+pub struct Script {
+    contracts: Vec<Vec<u8>>,
+    vm_kind: VMKind,
+    vm_config: VMConfig,
+    protocol_version: ProtocolVersion,
+    profile: ProfileData,
+    contract_cache: Option<Box<dyn CompiledContractCache>>,
+
+    initial_state: Option<State>,
+    steps: Vec<Step>,
+}
+
+pub struct Step {
+    contract: Contract,
+    method: String,
+    vm_context: VMContext,
+    promise_results: Vec<PromiseResult>,
+    repeat: u32,
+}
+
+pub struct ScriptResults {
+    pub outcomes: Vec<(Option<VMOutcome>, Option<VMError>)>,
+    pub state: MockedExternal,
+    pub profile: ProfileData,
+}
+
+impl Default for Script {
+    fn default() -> Self {
+        Script {
+            contracts: Vec::new(),
+            vm_kind: VMKind::default(),
+            vm_config: VMConfig::default(),
+            protocol_version: ProtocolVersion::MAX,
+            profile: ProfileData::new_disabled(),
+            contract_cache: None,
+
```

Comment:
Why extra space?

Reply:
No good reason really, removed it! 

My original thinking was to separate global readonly "static" info like `VmKind` from mutable things likes state, but that clearly won't survive the first refacroring of this code